src/hooks/useWalletModal.tsx

```diff
@@ -28,6 +29,22 @@ const WalletModal = ({ queryRef }: ModalProps) => {
   );
 };
 
+const Container = styled.div`
+  display: flex;
+  justify-content: center;
+  align-items: center;
+
+  // the height of the inner content with all wallet options listed.
+  // ensures the height of the modal doesn't shift
+  min-height: 360px;
+  height: 100%;
+
+  padding: 48px 24px;
+  @media only screen and ${breakpoints.tablet} {
+    padding: 0;
+  }
+`;
+
```

Comment:
used on the membership mint page

| Before | After |
| ----- | ----- |
| ![Screen Shot 2022-05-29 at 11 07 10 PM](https://user-images.githubusercontent.com/12162433/170910749-a00073b2-3d20-4688-a0d5-497d4c0d6859.png) | ![Screen Shot 2022-05-29 at 11 07 30 PM](https://user-images.githubusercontent.com/12162433/170910748-81594078-7bad-4bc6-83d9-9ad01c3d67e8.png) |